pom.xml

```diff
@@ -163,6 +163,14 @@
       <version>3.0.0</version>
     </dependency>
 
+    <!-- https://mvnrepository.com/artifact/org.apache.commons/commons-lang3 -->
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>3.12.0</version>
+    </dependency>
```

Comment:
StringUtils is helpful. It's up to @bglamadrid if he is ok with adding a new dependency.

Reply:
Yeah. Honestly I didn't like the null checking support of java 11 for String. I just want to avoid `str != null && !str.isEmpy()` vs `StringUtils.isNotBlank(str)` or `StringUtils.isNotEmpty(str)`